java/src/main/java/com/twisleton/gathering/dtos/Direction.java

```diff
@@ -1,5 +1,5 @@
 package com.twisleton.gathering.dtos;
 
 public enum Direction {
-    UP,DOWN,LEFT,RIGHT
+    NORTH,SOUTH,EAST,WEST,NORTHEAST,NORTHWEST,SOUTHEAST,SOUTHWEST
```

Comment:
Great!

Reply:
"DOWNLEFT" didn't roll off the tongue, so I switched these to cardinal direction names